common/Fabric-Install-Utilities.psm1

```diff
@@ -646,4 +695,6 @@ Export-ModuleMember -Function Add-ServiceUserToDiscovery
 Export-ModuleMember -Function Invoke-Sql
 Export-ModuleMember -Function Read-FabricInstallerSecret
 Export-ModuleMember -Function Get-ErrorFromResponse
-Export-ModuleMember -Function Invoke-ResetFabricInstallerSecret
\ No newline at end of file
+Export-ModuleMember -Function Invoke-ResetFabricInstallerSecret
+Export-ModuleMember -Function Test-DiscoveryHasBuildVersion
```

Comment:
Do we need to export the `Test-DiscoveryHasBuildVersion` function? Seems like right now it is only used internally in the module.